docs/workflows/package-composition.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'Package Composition'
+---
+
+Storybook is widely used by component libraries and design systems. Design system authors can automatically compose their design systems inside their consumer’s Storybooks. 
+
+For example, if you use a design system package, its stories can appear alongside your own. That makes it convenient to cross reference usage documentation without leaving Storybook.
+
+### For package consumers
+
+<div style="background-color:#F8FAFC">
+TODO: vet this item, supports link where should be pointed at
+</div>
+
+Composition happens automatically if the package [supports](locate-support-composition) it. When you install the package, Storybook will load its stories alongside your own. 
+
+![Package composition workflow](./package-composition.png)
+
+#### Configuring
+
+If you want to configure how the composed Storybook behaves, you can disable the `ref` element in your [`.storybook/main.js`](../configure/overview#configure-story-rendering)
+
+```js
+// .storybook/main.js
+
+module.exports = {
+  // your Storybook configuration
+  refs: {
+    'package-name': { disable: true }
+}
+```
+
+#### Changing versions
+
+<div style="background-color:#F8FAFC">
+TODO: vet this item, configuration link where should be pointed at
+</div>
+
+Change the version of the composed Storybook to see how the library evolves. This requires [configuration](locate-configuration) from the package author.
```

Comment:
Link to the "providing a version selection" below

Reply:
The link is updated per feedback.